python/ray/train/tests/test_mosaic_trainer.py

```diff
@@ -0,0 +1,138 @@
+import pytest
+
+import torch
+import torch.utils.data
+
+import torchvision
+from torchvision import transforms, datasets
+
+from torchmetrics.classification.accuracy import Accuracy
+from composer.core.evaluator import Evaluator
+from composer.models.tasks import ComposerClassifier
+import composer.optim
+from composer.loggers import InMemoryLogger
+from composer.algorithms import LabelSmoothing
+
+import ray
+from ray.air.config import ScalingConfig
+import ray.train as train
+from ray.air import session
+from ray.train.mosaic import MosaicTrainer
+
+
+mean = (0.507, 0.487, 0.441)
+std = (0.267, 0.256, 0.276)
+cifar10_transforms = transforms.Compose(
+    [transforms.ToTensor(), transforms.Normalize(mean, std)]
+)
+
+data_directory = "s3://air-example-data/cifar-10-python.tar.gz"
```

Comment:
Sorry I know we already asked you to change this, but we should be using a local directory here. we can just use `~/data`.

@jiaodong this directory is not where we are downloading the data from, but rather where we are writing the data to when downloading it from the public CIFAR host.

Reply:
Done!